### ISSUE TITLE
esp-csi/examples/esp-radar/console_test not working

### ISSUE THREAD

**Issue by rav2n-dev** [espressif/esp-csi#172]
Following the instructions doesn't give any waveform in the gui.

Please help as soon as possible, it is very urgent (for a humanitarian cause).

**Comment by MacChu0315-Espressif** [comment]
Does baud rate changing according #151 help?

**Comment by rav2n-dev** [comment]
> Does baud rate changing according to #151 help?

Made sure the baud rate was matching.
Still doesn't display any detail (waveform or logs or anything) unfortunately. It's all blank..

**Comment by MacChu0315-Espressif** [comment]
To better diagnose the issue, please monitor the ESP device using the command `idf.py monitor` and then enter the following two commands:

1. `radar --csi_output_type LLFT --csi_output_format base64`
2. `wifi_config --ssid "ROUTER_NAME" --password ROUTER_PASSWORD`

Additionally, could you please provide detailed descriptions of your testing environment? Include the type of development kit you are using, the version of the IDF, and any other relevant information. This information will greatly assist us in diagnosing and addressing your issue more effectively.

**Comment by q936505350** [comment]
![3J {7~%IOS7L~A8G4)7V_TO](https://github.com/espressif/esp-csi/assets/63338632/f9db2223-dc5e-4033-9cb1-e5c48ede6c11)
@MacChu0315-Espressif I have the same issue，and this is  the situation after I ran monitor……And, if I flash the "get-started\csi_recv" code, I can receive CSI data normally, but then I won't be able to use esp_csi_tools. I sincerely ask you how to solve this problem, please.

**Comment by MacChu0315** [comment]
Hi @q936505350: Firstly, regarding the image you provided, it's possible that the issue stems from an incorrect baud rate setting and we can't get enough info starting from the secondary bootloader. To assist you more effectively, could you please provide more detailed information? Secondly, the esp_csi_tool is utilized in conjunction with the console_test program. Lastly, based on the information provided, your issue does not seem to be closely related to the existing discussions. It is recommended that you initiate a new issue for further discussion.

**Comment by q936505350** [comment]
Thank you for your reply, I have resolved the issue.


936505350
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "espressif/esp-csi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月20日(星期一) 下午4:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [espressif/esp-csi] esp-csi/examples/esp-radar/console_test not working (AEGHB-619) (Issue #172)





 
Hi @q936505350: Firstly, regarding the image you provided, it's possible that the issue stems from an incorrect baud rate setting and we can't get enough info starting from the secondary bootloader. To assist you more effectively, could you please provide more detailed information? Secondly, the esp_csi_tool is utilized in conjunction with the console_test program. Lastly, based on the information provided, your issue does not seem to be closely related to the existing discussions. It is recommended that you initiate a new issue for further discussion.

**Comment by rav2n-dev** [comment]
> radar --csi_output_type LLFT --csi_output_format base64
> wifi_config --ssid "ROUTER_NAME" --password ROUTER_PASSWORD

Trying these out provides csi data logs and the controller seems to be running the flashed code (the sound it generates is not so low).

But the GUI doesn't display anything.. 

IDF vdersion: 5.0.2
CSI_RADAR Version: 0.2.0

OS: linux 

Examples to use:
https://github.com/espressif/esp-csi/blob/master/examples/get-started/csi_recv_router/README.md

and visualize using:
https://github.com/espressif/esp-csi/tree/master/examples/esp-radar/console_test

tried flashing csi_recv_router then console_test/tool/esp_csi_tool.py
also tried flashing console_test then console_test/tool/esp_csi_tool.py

basically followed the READMEs in csi_recv_router (as I only have one esp32) and console_test.

Also set the matching right baud rate.

**Comment by MacChu0315-Espressif** [comment]
Hi @rav2n-dev, can you provide the output from running the following two lines of code?


> To better diagnose the issue, please monitor the ESP device using the command `idf.py monitor` and then enter the following two commands:
> 
> 1. `radar --csi_output_type LLFT --csi_output_format base64`
> 2. `wifi_config --ssid "ROUTER_NAME" --password ROUTER_PASSWORD`
> 
> Additionally, could you please provide detailed descriptions of your testing environment? Include the type of development kit you are using, the version of the IDF, and any other relevant information. This information will greatly assist us in diagnosing and addressing your issue more effectively.
